examples/update/canisters/motoko/main.mo

```diff
@@ -0,0 +1,39 @@
+import Prim "mo:⛔";
+
+actor Update {
+  //#region Performance
+  type PerfResult = {
+    wasm_body_only: Nat64;
+    wasm_including_prelude: Nat64;
+  };
+
+  var perf_result: ?PerfResult = null;
+
+  public query func get_perf_result(): async ?PerfResult {
+    return perf_result;
+  };
+
+  func record_performance(start: Nat64, end: Nat64) : () {
+    perf_result := ?{
+      wasm_body_only = end - start;
+      wasm_including_prelude = Prim.performanceCounter(0);
+    };
+  };
+  //#endregion
+
+  var currentMessage = "";
+
+  // query is a reserved word and therefore cannot be a function name
+  public query func query1() : async Text {
```

Comment:
The Azle and Motoko names for this function should be the same, perhaps something like `get_current_message` would be better than `query1`?